backend/tracim_backend/lib/webdav/design.py

```diff
@@ -124,6 +126,16 @@
     'copy' : 'Item copied',
 }
 
+THREAD_MESSAGE = """
+  <p>
+    {posting_time},{comment_owner} wrote :
+  </p>
+  <p style="padding-left: 1em;">
```

Comment:
@basilelegal why dont you use the same design as ou do for other messages: `<blockquote style="border-left: solid 2px #999; margin-left: 1em; padding-left: 1em;">` ?